tests/includes/server-timing/load-tests.php

```diff
@@ -96,74 +108,104 @@ public function test_perflab_sanitize_server_timing_setting( $unsanitized, $expe
 	}
 
 	public function data_perflab_sanitize_server_timing_setting() {
```

Comment:
Changes in this data provider due to https://github.com/WordPress/performance/pull/1188/files#r1589523090

Reply:
And https://github.com/WordPress/performance/pull/1188/files#r1589525953